install/generator/03-syndesis-ui.yml.mustache

```diff
@@ -105,6 +105,7 @@
         "apiBase": "https://${ROUTE_HOSTNAME}",
         "apiEndpoint": "/api/v1",
         "title": "Syndesis",
+        "openshiftConsoleProjectURL": "${OPENSHIFT_CONSOLE_URL}/project/${OPENSHIFT_PROJECT}",
```

Comment:
@chirino If we put in two arguments `consoleUrl` and `project` which we individually populate with `${OPENSHIFT_CONSOLE_URL}` and `${OPENSHIFT_PROJECT}1`, then in the UI we could combine both, but only when `consoleUrl` is set, then we could make `${OPENSHIFT_CONSOLE_URL}` optional (so that the link doesnt show up when the base url is not set).

Let me update the PR to adapt to this, i think this is a good compromise between avoiding mandatory parameter bloat and adding this link. You have the choice when installing the templates.